src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+let collection = [];
+const form = document.getElementById('addTodo');
+
+function ShowList(arr) {
+  const listToDo = arr.map((b) => `
+    <ul class="testList1" draggable="true">
+          <li><input type="checkbox" id='${b.id}' value='${b.complete}' class="checkboX">
+          <p>${b.title}</p>
+          </li>
+          <i class=" fa fa-ellipsis-v" id='${b.id}'></i>
+          <i class="fa fa-trash-o" id='${b.id}'></i>          
+      </ul>
+     `).join('');
+  document.getElementById('showListItem').innerHTML = `${listToDo}`;
+}
+
+function addList() {
+  const todo = {
+    id: collection.length,
+    title: document.getElementById('todotitle').value,
+    complete: false,
+  };
+  collection.push(todo);
+  localStorage.setItem('todoList', JSON.stringify(collection));
+  if (collection.length > 0) {
+    ShowList(collection);
+  }
+  form.reset();
+}
+
+window.addEventListener('load', () => {
+  const dataGet = localStorage.getItem('todoList');
+  const data = JSON.parse(dataGet);
+  if (data) {
+    collection = data;
+  }
+  if (collection.length > 0) {
+    ShowList(collection);
+  }
+});
+
+form.addEventListener('submit', (ev) => {
```

Comment:
- [ ] Please make sure it has a function that iterates over the tasks array and populates the HTML list item element for each task.

```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```

Reply:
thanks for your review but I used a function to iterate over the array that is show list and I used a map inside the function. And it is working

Reply:
If you feel it's working please accept my apologies. You can request another review was you feel you are done with all the changes that can implement. Thank you for understanding

Reply:
Thank you for your reply. I will update my project with required changes and submit again